test/common/pixeldiff.html

```diff
@@ -0,0 +1,598 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Cockpit Integration Tests - Pixel diffs</title>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js" type="text/javascript"></script>
```

Comment:
Let's eliminiate this in a follow-up. The operations here are simple enough to do them with the standard browser DOM API.

Reply:
The whole pixeldiff.html file could be redone from scratch, tbh.